workflow/main.py

```diff
@@ -0,0 +1,213 @@
+
+    # dir name set manually
+    DIRNAME = r"C:\Users\ajshe\OneDrive\Documents\Comp_bio\Cytoscape_v3.7.1"
+    CYTOSCAPE = 'Cytoscape.exe'
+
+    if OUTPUT_FILE.lower().endswith(".sif"):
+        print("file has valid extention")
+    else:
+        print("invalid extension on output file: ", OUTPUT_FILE)
+        sys.exit()
+
+    # get absolute paths if nessesary
+    OUTPUT_FILE = os.path.abspath(OUTPUT_FILE)
+    STYLE_FILE = os.path.abspath(STYLE_FILE)
+    print("---absolute output path: ", OUTPUT_FILE)
+    print("---absolute style path: ", STYLE_FILE)
+
+    # * Gather data used to generate node anotations
+    # Use the version with the header line
+    pepMapFile = os.path.join(tps_in_dir, 'peptide-mapping.tsv')
+    pepFirstFile = os.path.join(tps_in_dir, 'p-values-first.tsv')
+    pepPrevFile = os.path.join(tps_in_dir, 'p-values-prev.tsv')
+    print(pepMapFile)
+    print(pepFirstFile)
+    print(pepPrevFile)
+
+    # Use the version for which log2 fold change has been precomputed
+    timeSeriesFile = os.path.join(
+        baseDir, 'data', 'timeseries', 'log2FoldChange011215.txt')
+    #timeSeriesFile = r"C:\\Users\\ajshe\\Anaconda3\\envs\\py2cyto\\tps\\data\\timeseries\\log2FoldChange011215.txt"
+    print(timeSeriesFile)
+
+    windowsFile = os.path.join(baseDir, 'workflow', 'activity-windows.tsv')
+    networkFile = os.path.join(baseDir, 'workflow', 'output.sif')
+    print(windowsFile)
+    print(networkFile)
+
+    # Use the same EGFR gold standard for Olsen 2006 and Ale's data
+    goldStandardFile = os.path.join(
+        baseDir, 'data', 'resources', 'eight-egfr-reference-all.txt')
+
+    styleTemplateFile = os.path.join(
+        baseDir, 'notebooks', 'tps_style_template.xml')
+
+    out_dir = os.path.join(baseDir, 'results', str(
+        today) + '-and-wolf-yadlin-TPS-cytoscape')
+    if not os.path.exists(out_dir):
+        os.makedirs(out_dir)
+        print('Created {}'.format(out_dir))
+    outFile = os.path.join(
+        out_dir, 'wolf-yadlin-cytoscape-annotations-' + str(today) + '.txt')
+    outStyleFile = os.path.join(out_dir, 'tps_style.xml')
+
+    # * run Utility function
+    pvalThresh = 0.01  # Same threhsold used in TPS
+    logTransform = False
+    pepsPerProt = PrepTemporalCytoscapeTPS(pepMapFile, timeSeriesFile, pepFirstFile,
+                                           pepPrevFile, windowsFile, networkFile,
+                                           goldStandardFile, pvalThresh, logTransform, styleTemplateFile,
+                                           outFile,
+                                           outStyleFile,
+                                           addZero=True)  # don't provide logDefault
+
+    # * get annotations file
+    ANNOTATIONS_FILE = outFile
+    print("---absolute annotations path: ", ANNOTATIONS_FILE)
+
+    # * Prepare data and directories
+    data_delim = pd.read_csv(ANNOTATIONS_FILE, delimiter='\t').drop('Unnamed: 19', 1)
```

Comment:
What is `Unnamed: 19`?

Reply:
This was a default column that was added as a result of reading the annotations file as above. This was a simple solution at the time, but I will need to look at this further.